controllers/clusteroperator.go

```diff
@@ -112,7 +128,7 @@ func setStatusCondition(conditionType osconfigv1.ClusterStatusConditionType,
 	}
 }
 
-// getOperandVersions returns current version
+// getOperandVersions returns the operand version
 func getOperandVersions() []osconfigv1.OperandVersion {
 	operandVersions := []osconfigv1.OperandVersion{}
 	if releaseVersion := os.Getenv("RELEASE_VERSION"); len(releaseVersion) > 0 {
```

Comment:
Should we report an error if we can't figure out our version?

Reply:
+1. I think this is env value required and will be passed to the CBO deployment. Nevertheless, we should do something about it if it goes missing.